apps/data/database/populatedb.sql

```diff
@@ -1,9 +1,9 @@
 INSERT INTO users (username, email, password, name, bio, profile_pic_url, verified) VALUES
-('user', 'user@uvic.ca', 'Testing1!', 'User', 'Hi! Im a verified user!', 'https://api.dicebear.com/8.x/adventurer/svg?seed=Jasper', TRUE),
+('deleted_user', 'deleted@martletplace.com', 'iGKT2wOGvPkn5s6U', 'Deleted User', '', 'https://api.dicebear.com/8.x/adventurer/svg?seed=Jasper', TRUE),
```

Comment:
Ah, understood. Still confused why we aren't just allowing the cascade delete to do its thing though. Do we really want that stuff staying around?

Reply:
Yes, we don't want messages or reviews (primarily messages) to be removed when a user is deleted